src/components/Footer.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable jsx-a11y/alt-text */
+import React from 'react'
+import '../style/Footer.css'
+function Footer() {
```

Comment:
Make a file called constants.js and add all the strings found here to it then export them.

And in the footer, import them.

The constants.js file will look something like this:

export const getInTouch="Get In Touch"
export const ourShop="Our Shop"